pkgs/servers/jetbrains/youtrack.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name = "youtrack-${version}";
+  version = "2018.1.40341";
+
+  src = fetchurl {
+    url = "https://download.jetbrains.com/charisma/${name}.jar";
+    sha256 = "1qk643yr525qrw21ly1w29fmkz4gv0k44ai6f6v2gqdn4v2w1zql";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  unpackPhase = "true";
+
+  installPhase = ''
+    jar=$out/share/java/${name}.jar
+    install -Dm444 $src $jar
+    makeWrapper ${jre}/bin/java $out/bin/youtrack --add-flags "\$YOUTRACK_JVM_OPTS -jar $jar"
+  '';
+
+  meta = {
```

Comment:
`meta = with stdenv.lib; {` would allow to omit `stdenv.lib` inside the attrset.